vars/buildPlugin.groovy

```diff
@@ -225,7 +225,7 @@ static List<Map<String, String>> getConfigurations(Map params) {
  * Includes testing Java 8 and 11 on the newest LTS.
  */
 static List<Map<String, String>> recommendedConfigurations() {
-    def recentLTS = "2.164.1"
+    def recentLTS = "2.176.3"
```

Comment:
Months later now... 
Perhaps it we can change this to a currently recent LTS? 
```suggestion
    def recentLTS = "2.204.6"
```

Reply:
Or 2.222.3?

Reply:
```suggestion
    def recentLTS = "2.222.3"
```